core/dictgen/src/rootcling_impl.cxx

```diff
@@ -2136,6 +2136,10 @@ void AddPlatformDefines(std::vector<std::string> &clingArgs)
    clingArgs.push_back(platformDefines);
    snprintf(platformDefines, 64, "-DG__VISUAL=%ld", (long)_MSC_VER);
    clingArgs.push_back(platformDefines);
+#if defined(_WIN64) && defined(_DEBUG)
+   snprintf(platformDefines, 64, "-D_ITERATOR_DEBUG_LEVEL=0"); //, (long)_ITERATOR_DEBUG_LEVEL);
```

Comment:
Does that fix: "the I/O is still broken in debug mode, due to the special iterators on x64," ?

Reply:
No, it's just a workaround. The problem is that one can only link the ROOT libraries in debug mode if they use the same flag with their software. Even worse, the externals have to be built with the same flag (e.g. I had the issue with Pythia8)!
And you are the only one able to fix the issue!

Reply:
Ok.